client/components/ProgressBar.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import style from '../style.css';
+
+const ProgressBar = (props) => {
```

Comment:
You could destructure in the function header:
```
const ProgressBar = ({ goal, fill }) => {
```

Reply:
True. I personally find it more readable to destructure within the function body. Thanks for the review.